wagtailio/project_styleguide/templates/patterns/pages/home/home_page.html

```diff
@@ -9,9 +9,9 @@
     {% include "patterns/components/hero/hero.html" with value=page classes="hero--home" %}
 
     <div class="grid">
-        <div class="code-snippet{% if not page.cta_one or not page.cta_two %} code-snippet--bottom-space{% endif %}" data-code-snippet>
-            <pre data-code-to-copy>pip install wagtail</pre>
-            <button class="code-snippet__button" aria-label="Copy the command pip install wagtail" data-copy-code-button>
+        <div class="code-snippet{% if page.call_to_action|length < 2 %} code-snippet--bottom-space{% endif %}" data-code-snippet>
```

Comment:
since `code_block` can be blank, you want to wrap this in an if.

Reply:
done in 34198484